java/src/test/java/ai/rapids/cudf/ReductionTest.java

```diff
@@ -430,6 +431,7 @@ private static void assertEqualsDelta(ReductionAggregation op, Scalar expected,
     }
   }
 
+  @Tag("noSanitizer")
```

Comment:
Since it's passing under CUDA11 Sanitizer, maybe we can call the tag accordingly `cuda11Sanitizer`. If the user has access to both 11 and 12 on CI they can choose accordingly?

Reply:
Sure, although when the tag was created it wasn't expected the sanitizer would run so differently across CUDA versions.  Note that this is just a short-term fix to unblock CI, we don't have plans to run it in a split form long-term unless it's a bug in compute-sanitizer itself that we would need to work around.